libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid.service.ts

```diff
@@ -84,7 +90,10 @@ function dateComparator(date1: any, date2: any): number {
   return date1value ? 1 : -1;
 }
 
-function getValidatorCellRendererSelector(component: string, fallback?: any) {
+function getValidatorCellRendererSelector(
+  component: string,
+  fallback?: CellRendererSelectorResult | undefined
```

Comment:
Since we're using the `?` we can probably remove the redundant `| undefined`.

Reply:
Updated.